vpr/src/route/route_budgets.cpp

```diff
@@ -170,13 +179,16 @@ void route_budgets::allocate_slack_using_weights(vtr::vector_map<ClusterNetId, f
     iteration = 0;
     max_budget_change = 900e-12;
 
-    /*This allocates long path slack and increases the budgets*/
-    while ((iteration > 3 && max_budget_change > 800e-12) || iteration <= 3) {
+    //pass in original timing_info to use for total path delay calculations. This ensures the same total path delay is used throughout the RCV iterations.
+    orig_timing_info = perform_sta (delay_max_budget);
+
+    /*This allocates long path slack and increases the budgets. Stopping conditions are set differently from RCV paper because they showed better results*/
+    while ((iteration > 3 && max_budget_change > 1e-12) || iteration <= 3) {
```

Comment:
Could this condition be simplified to:
```
while(max_budget_change > 1e-12)
```

It seems like requiring a minimum of 3 iterations is unnecessary (unless I'm missing something).